README.md

```diff
@@ -1 +1,125 @@
-# ansible-role-template
\ No newline at end of file
+# Ansible Role Template
+
+![Pipeline Status](https://github.com/mentoriaiac/template-role-ansible/actions/workflows/ci.yml/badge.svg)
+
+This repository provides a base scaffolding template to use as a quick start of
+all Ansible roles at @Appsilon. Includes best practices that should always be
+used, like linter, tests, CI, etc.
+
+## How to Use
+
+1. Click "[Use this
+   template](https://github.com/Appsilon/ansible-role-template/generate)" to
+   create a new repository from it.
+1. Update [`./meta/main.yml`](./meta/main.yml) with updated repository and role names.
+1. Update [`./role_README.md`](./role_README.md) with updated to repository and role name.
+1. Rename `./role_README.md` to `README.md`.
+1. Start coding! You're free to change anything.
+
+## Development
+
+### Dependencies
+
+To execute tests locally, it is necessary to install the following
+dependencies:
+
+* [Python](https://www.python.org/downloads/)
+* [Molecule](https://molecule.readthedocs.io/en/latest/installation.html)
+
+### Preparing the Environment
+
+Create a Python environment:
+
+```bash
+python3 -m venv .venv
+```
+
+Activate the environment:
+
+```bash
+source .venv/bin/activate
+```
+
+Install molecule (and its dependencies) inside the environment:
+
+```bash
+python3 -m pip install -r requirements.txt
+```
+
+### Running
+
+```bash
+molecule test
+```
+
+To perform quick test after some modification:
+
+```bash
+molecule create
+molecule converge
+molecule verify
+```
+
```

Comment:
Can we add molecule login here? I use quite often to test the running containers